### PR TITLE
feat: tagline-off-*-v3.json

### PR DESCRIPTION
**Pull request by teolemon**
### What
- Files for the 2nd iteration of the tagline format (v3 because we have v1 and v2 of the app) 

### Related
- https://github.com/openfoodfacts/smooth-app/pull/5350

### CODE REVIEW

File: html/resources/files/tagline-off-android-v3.json [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+               "title":"My title",
+               "message":"My message",
+               "button_label":"My button",
+               "image":{
+                  "url":"https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-en.svg",
+                  "width":0.2
+               }
+            },
+            "fr":{
+               "url":"https://translate.openfoodfacts.org/project/openfoodfacts/fr",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:58]:
```suggestion
               "url":"https://fr.openfoodfacts.org/nutriscore-v2",
```

File: html/resources/files/tagline-off-android-v3.json [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+            "title_indicator_color":"#EB5757",
+            "message_background":"#000000",
+            "message_text_color":"#FFFFFF",
+            "button_background":"#000000",
+            "button_text_color":"#FFFFFF",
+            "content_background_color":"#219653"
+         },
+         "translations":{
+            "default":{
+               "url":"https://translate.openfoodfacts.org/project/openfoodfacts/en",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:48]:
```suggestion
               "url":"https://world.openfoodfacts.org/nutriscore-v2",
```

File: html/resources/files/tagline-off-android-v3.json [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+               "title":"My title",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:49]:
```suggestion
               "title":"It's all new !",
```

File: html/resources/files/tagline-off-android-v3.json [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+               "button_label":"My button",
+               "image":{
+                  "url":"https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-en.svg",
+                  "width":0.2
+               }
+            },
+            "fr":{
+               "url":"https://translate.openfoodfacts.org/project/openfoodfacts/fr",
+               "title":"C’est tout nouveau !",
+               "message":"Le **Nouveau Calcul** du NutriScore est disponible dans l’application",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:60]:
```suggestion
               "message":"The **New Calculation** of NutriScore is available in the application",
```

File: html/resources/files/tagline-off-android-v3.json [hunk 5]
```diff
@@ -0,0 +1,89 @@
...
+            "message_text_color":"#FFFFFF",
+            "button_background":"#000000",
+            "button_text_color":"#FFFFFF",
+            "content_background_color":"#219653"
+         },
+         "translations":{
+            "default":{
+               "url":"https://translate.openfoodfacts.org/project/openfoodfacts/en",
+               "title":"My title",
+               "message":"My message",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:50]:
```suggestion
               "message":"The **New Calculation** of NutriScore is available in the application",
```

File: html/resources/files/tagline-off-android-v3.json [hunk 6]
```diff
@@ -0,0 +1,89 @@
...
+               "button_label":"My button",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:51]:
```suggestion
               "button_label":"Learn more",
```

File: html/resources/files/tagline-off-android-v3.json (hunk 5 above)
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:50]:
```suggestion
               "title":"Bring food transparency to your language",
               "message":"Translate the app, website and more to help others make informed food choices",
```

File: html/resources/files/tagline-off-android-v3.json [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+               }
+            },
+            "fr":{
+               "url":"https://translate.openfoodfacts.org/project/openfoodfacts/fr",
+               "title":"Rendez la transparence alimentaire accessible dans votre langue",
+               "message":"Traduisez l'application, le site web et plus encore pour aider les autres à faire des choix alimentaires éclairés",
+               "button_label":"Rejoindre l'équipe de traducteurs"
+
+               "image":{
+                  "url":"https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-fr.svg",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:64]:
```suggestion
                  "url":"https://world.openfoodfacts.org/images/misc/tagline-images/worldwide.png",
```

Comment (teolemon) [reply html/resources/files/tagline-off-android-v3.json:64]: https://github.com/openfoodfacts/openfoodfacts-web/pull/590/files#diff-b0dc30243a5a7e35f56a704d6bb0e006d14e74ad530118b8021c6cbe3c348a58

File: html/resources/files/tagline-off-android-v3.json [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+            "content_background_color":"#219653"
+         },
+         "translations":{
+            "default":{
+               "url":"https://translate.openfoodfacts.org/project/openfoodfacts/en",
+               "title":"Bring food transparency to your language",
+               "message":"Translate the app, website and more to help others make informed food choices",
+               "button_label":"Join the translators team",
+               "image":{
+                  "url":"https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-en.svg",
```
[the remark this answers is not in the document]
Comment (teolemon) [reply html/resources/files/tagline-off-android-v3.json:53]: https://github.com/openfoodfacts/openfoodfacts-web/pull/590/files#diff-b0dc30243a5a7e35f56a704d6bb0e006d14e74ad530118b8021c6cbe3c348a58

File: html/resources/files/tagline-off-android-v3.json [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+         "style":{
+            "title_background":"#000000",
+            "title_text_color":"#FFFFFF",
+            "title_indicator_color":"#EB5757",
+            "message_background":"#000000",
+            "message_text_color":"#FFFFFF",
+            "button_background":"#000000",
+            "button_text_color":"#FFFFFF",
+            "content_background_color":"#219653"
+         },
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:45]:
```suggestion
```

File: html/resources/files/tagline-off-android-v3.json [hunk 10]
```diff
@@ -0,0 +1,347 @@
...
+        "width": 0.2,
+        "alt": "Het Nutri-Score A-logo"
+      }
+    },
+    "pt": {
+      "title": "É novo!",
+      "message": "O **Novo Cálculo** do Nutri-Score está disponível no aplicativo",
+      "button_label": "Saiba mais",
+      "image": {
+        "url": "https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-pt.svg",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:55]:
```suggestion
        "url": "https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-en.svg",
```

File: html/resources/files/tagline-off-android-v3.json [hunk 11]
```diff
@@ -0,0 +1,347 @@
...
+        "width": 0.2,
+        "alt": "O logotipo Nutri-Score A"
+      }
+    },
+    "it": {
+      "title": "È nuovo!",
+      "message": "Il **Nuovo Calcolo** del NutriScore è disponibile nell'app",
+      "button_label": "Scopri di più",
+      "image": {
+        "url": "https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-it.svg",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:65]:
```suggestion
        "url": "https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-en.svg",
```

File: html/resources/files/tagline-off-android-v3.json [hunk 12]
```diff
@@ -0,0 +1,347 @@
...
+        "width": 0.2,
+        "alt": "Il logo NutriScore A"
+      }
+    },
+    "es": {
+      "title": "¡Es nuevo!",
+      "message": "El **Nuevo Cálculo** del Nutri-Score está disponible en la aplicación",
+      "button_label": "Más información",
+      "image": {
+        "url": "https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-es.svg",
```
Comment (teolemon) [inline html/resources/files/tagline-off-android-v3.json:75]:
```suggestion
        "url": "https://static.openfoodfacts.org/images/attributes/dist/nutriscore-a-new-en.svg",
```

**Review by alexgarel — APPROVED** [review]
LGTM

**Pull request merged**